src/components/refactor/inputs/utils.js

```diff
@@ -0,0 +1,35 @@
+import { IconButton, InputAdornment } from '@mui/material';
+import ClearIcon from '@mui/icons-material/Clear';
+
+export const getAdornmentInputProps = ({ position, text }) => {
+    const adornmentProp = position + 'Adornment';
+    return ({ value, isFocused }) => ({
+        InputProps: {
+            [adornmentProp]:
+                (value !== undefined && value !== '') || isFocused ? (
+                    <InputAdornment position={position}>{text}</InputAdornment>
+                ) : (
+                    <></>
+                ),
+        },
+        sx: { input: { textAlign: position } },
+    });
+};
+
+export const getClearAdornmentInputProps = ({ position }) => {
```

Comment:
makeClearAdornmentCallback ?

Reply:
function has been removed in favor of the already existing ones